lib/secret_code.rb

```diff
@@ -1,8 +1,11 @@
 require './output'
+require 'pry'
```

Comment:
Don't require pry in PRs

Reply:
Er I mean, don't require pry after you're done, get rid of it.

Trick: `require 'pry'; binding.pry` is a one liner to initiate pry w/o requiring it at the top of the file so you don't forget to remove it